### PR TITLE
[nxp][examples][contact-sensor][zap] Update zap files

### PR DESCRIPTION
**Pull request by andrei-menzopol**
* Update cluster version
     * Update AccessControl feature map
     * Add OTA Requestor device type to EP0

### CODE REVIEW

**Review by doru91 — APPROVED** [review]

**Review by tcarmelveilleux — APPROVED** [review]
Please ensure your sample passes TC_DeviceBasicComposition.py and TC_DeviceConformance.py tests before merging.

File: examples/contact-sensor-app/nxp/zap-lit/contact-sensor-app.matter [hunk 1]
```diff
@@ -1747,7 +1751,7 @@ endpoint 0 {
     callback attribute generatedCommandList;
     callback attribute acceptedCommandList;
     callback attribute attributeList;
-    ram      attribute featureMap default = 0;
+    ram      attribute featureMap default = 1;
```
Comment (tcarmelveilleux) [inline examples/contact-sensor-app/nxp/zap-lit/contact-sensor-app.matter:1754]: Please ensure your sample passes TC_DeviceBasicComposition.py and TC_DeviceConformance.py tests before merging.

Comment (cecille) [reply examples/contact-sensor-app/nxp/zap-lit/contact-sensor-app.matter:1754]: Is the contact sensor app actually using the ACL extension?

Comment (andrei-menzopol) [reply examples/contact-sensor-app/nxp/zap-lit/contact-sensor-app.matter]: We have tested the respective tests and all pass.
Also, I have disabled the ACL extension.
Thank you

**Pull request merged**